src/controllers/itemdetailpage.js

```diff
@@ -463,19 +463,26 @@ define(["loading", "appRouter", "layoutManager", "userSettings", "connectionMana
         var imgUrl;
         var screenWidth = screen.availWidth;
         var hasbackdrop = false;
+        var backdropWrapper = page.querySelector(".backdropWrapper");
         var itemBackdropElement = page.querySelector("#itemBackdrop");
+        var btnBackdropShow = page.querySelector(".btnBackdropShow");
```

Comment:
I still think we should move this option (and add a few more options) to the user preferences or something to avoid cluttering the page.

Reply:
that's great for know i will remove it the btn show/hide from the deatilpage

Reply:
https://github.com/jellyfin/jellyfin-web/pull/611/commits/e0d6686bb48204961b0e8ffd97f32a1301b0a424